themes/getting-started/setting-up-your-local-environment.md

```diff
@@ -30,7 +30,7 @@ Open a command line on your (empty) working directory, then:
 
 Using git you can choose your PrestaShop version:
 ```bash
-git checkout 8.0
+git checkout 8.0.0
```

Comment:
nice catch. We can also replace it with `8.1.x` branch ;-)

Reply:
Done :)